airflow/providers/google/cloud/operators/bigquery.py

```diff
@@ -292,6 +301,9 @@ def _submit_job(
         """Submit a new job and get the job id for polling the status using Trigger."""
         configuration = {"query": {"query": self.sql, "useLegacySql": self.use_legacy_sql}}
 
+        if self.encryption_configuration is not None:
+            configuration["query"]["destinationEncryptionConfiguration"] = self.encryption_configuration
+
```

Comment:
Suggestion: I think you can extract lines 302-305 into a separate method. Something like create_job_configuration()